boefjes/boefjes/plugins/kat_abuseipdb/main.py

```diff
@@ -0,0 +1,34 @@
+from ipaddress import ip_address
+from os import getenv
+
+import httpx
+
+REQUEST_TIMEOUT = 60
+
+
+def run(boefje_meta: dict) -> list[tuple[set, bytes | str]]:
+    """Make request to AbuseIPDB."""
+
+    if boefje_meta["arguments"]["input"]["network"]["name"] != "internet":
+        return [({"info/boefje"}, "Skipping non-internet network")]
```

Comment:
We already skip private IP addresses below, so this check is not necessary and can be removed. We are trying to remove hardcoded network name "internet" is hardcoded, we should introduce more places.

Reply:
Fixed in 77e393c0cc57623657f315cba6600c0a35c419ad